### PR TITLE
zh-translation:/docs/reference/glossary/multi-mesh.md

### PR DESCRIPTION
**Pull request by niceforbear**
ref: https://github.com/servicemesher/istio-official-translation/issues/1265

### CODE REVIEW

**Comment by tanjunchen** [comment]
please fix conclicts.Thanks

**Comment by tanjunchen** [comment]
@niceforbear 请解决下冲突，或者重新提交，目前未翻译。
相关帮助可以在钉钉群询问

File: content/zh/docs/reference/glossary/multi-mesh.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
-title: Multi-Mesh
+title: Multi-mush
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/multi-mesh.md]: mush -> mesh

**Comment by tanjunchen** [comment]
@niceforbear  The file has been translated . So Close This pr . Thanks your job!
@rootsongjc

**Pull request closed without merging**